include/envoy/server/options.h

```diff
@@ -69,6 +69,11 @@ class Options {
   virtual const std::string& adminAddressPath() PURE;
 
   /**
+   * @return const std::string& the local address IP version.
+   */
+  virtual const std::string& localAddressIpVersion() PURE;
```

Comment:
Would it be better to just do the string conversion to enum inside this function? Do we need the conversion elsewhere?